ecl/resource_ecl_rca_user_v1_test.go

```diff
@@ -101,14 +102,17 @@ func testAccCheckRCAV1UserExists(n string, user *users.User) resource.TestCheckF
 	}
 }
 
-const testAccRCAV1UserBasic = `
-resource "ecl_rca_user_v1" "user_1" {
+var randomName = acctest.RandStringFromCharSet(16, acctest.CharSetAlphaNum)
```

Comment:
This naming will cause name duplication with other files. But I do not want to use long name for this variable. Can you check how other terraform providers do that?

Reply:
It turned out the variable names in other files are relatively longer like this:

`import_ecl_dns_recordset_v2_test.go`:10
```
	zoneName := randomZoneName()
	resourceName := "ecl_dns_recordset_v2.recordset_1"
```

But as you mentioned above I can see we are going to duplicate `randomName` and `resourcePath` over and over.
So, can I use the names such as `randName`, `resPath`, or something?